source/amazon/services/supported-services/inspector.rst

```diff
@@ -1,22 +1,73 @@
+
 .. _amazon_inspector:
 
-Amazon Inspector
-================
+Amazon Inspector Classic
+========================
+
+`Amazon Inspector Classic <https://docs.aws.amazon.com/inspector/v1/userguide/inspector_introduction.html>`_ is an automated security assessment service that helps improve the security and compliance of applications deployed on AWS. Amazon Inspector Classic automatically assesses applications for exposure, vulnerabilities, and deviations from best practices. After performing an assessment, Amazon Inspector Classic produces a detailed list of security findings prioritized by level of severity. These findings can be reviewed directly or as part of detailed assessment reports which are available via the Amazon Inspector console or API.
 
-`Amazon Inspector <https://aws.amazon.com/inspector/>`_ is an automated security assessment service that helps improve the security and compliance of applications deployed on AWS. Amazon Inspector automatically assesses applications for exposure, vulnerabilities, and deviations from best practices. After performing an assessment, Amazon Inspector produces a detailed list of security findings prioritized by level of severity. These findings can be reviewed directly or as part of detailed assessment reports which are available via the Amazon Inspector console or API.
+.. note::
+   Wazuh does not yet support the `new Amazon Inspector module <https://aws.amazon.com/inspector/>`_, but it has support for the `Amazon Inspector Classic service <https://docs.aws.amazon.com/inspector/v1/userguide/inspector_introduction.html>`_.
 
 Amazon configuration
 --------------------
 
-Amazon Inspector does not need to store logs into a bucket, like the other AWS services.
-The inspector works as a searcher, so it retrieves information using the AWS API and provides an agent that analyzes it.
+#. To start using Amazon Inspector Classic, go to the Amazon Web Services management console and search for the Inspector service. Once there, click on the left side menu.
+
+    .. thumbnail:: ../../../images/aws/aws-inspector-overview.png
+	:align: center
+	:width: 100%
+
+#. Click on ``Switch to Inspector Classic``.
+
+    .. thumbnail:: ../../../images/aws/aws-inspector-side-menu.png
+	:align: center
+	:width: 100%
+
+#. Click on ``Get started``.
+
+    .. thumbnail:: ../../../images/aws/aws-inspector-get-started.png
+	:align: center
+	:width: 100%
+
+#. Click on ``Advanced setup``.
+
+    .. thumbnail:: ../../../images/aws/aws-inspector-advanced-setup.png
+	:align: center
+	:width: 100%
+
+#. Choose a name for the assessment target and select if you are going to include all EC2 instances in the scans or if you prefer to filter them based on a tag by enabling or disabling the ``Include all EC2 instances in this AWS account and region`` checkbox.
+
+   If you prefer to install the agents by hand, disable the ``Install the Amazon Inspector Agent on all EC2 instances in this assessment target`` checkbox.
```

Comment:
Please rework this step. Split it into several steps or a nested list if necessary

Reply:
Fixed in [efa9413](https://github.com/wazuh/wazuh-documentation/pull/4797/commits/efa94139a414b48ec143b9d3e22578a6f83217bd)!